src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.ts

```diff
@@ -129,32 +130,17 @@ export class GqrRqgViewResponsesComponent extends ResponsesInstructorCommentsBas
             return false;
           }
 
-          let shouldDisplayBasedOnSection: boolean = true;
-          if (this.section) {
-            switch (this.sectionType) {
-              case InstructorSessionResultSectionType.EITHER:
-                shouldDisplayBasedOnSection =
-                    response.giverSection === this.section || response.recipientSection === this.section;
-                break;
-              case InstructorSessionResultSectionType.GIVER:
-                shouldDisplayBasedOnSection = response.giverSection === this.section;
-                break;
-              case InstructorSessionResultSectionType.EVALUEE:
-                shouldDisplayBasedOnSection = response.recipientSection === this.section;
-                break;
-              case InstructorSessionResultSectionType.BOTH:
-                shouldDisplayBasedOnSection =
-                    response.giverSection === this.section && response.recipientSection === this.section;
-                break;
-              default:
-            }
-          }
+          const shouldDisplayBasedOnSection: boolean
+            = this.feedbackResponsesService.isFeedbackResponsesDisplayedOnSection(this.section,
+                this.sectionType, response);
```

Comment:
```suggestion
          const shouldDisplayBasedOnSection: boolean = this.feedbackResponsesService
            .isFeedbackResponsesDisplayedOnSection(this.section, this.sectionType, response);
```
Might be worthwhile having this as two lines instead of three, feels cleaner in my opinion